sage-ios/SAGE/SAGE/AdminOperations.swift

```diff
@@ -12,11 +12,15 @@ import SwiftKeychainWrapper
 
 class AdminOperations {
     
-    static func loadMentors(completion: (([User]) -> Void), failure: (String) -> Void){
-
+    static func loadMentors(filter filter: [String: AnyObject]? = nil, completion: (([User]) -> Void), failure: (String) -> Void){
+        var params = [String: AnyObject]()
+        if let filter = filter as [String: AnyObject]! {
+            for (key, value) in filter {
+                params[key] = String(value)
+            }
+        }
         let manager = BaseOperation.manager()
         if let currentSemester = KeychainWrapper.objectForKey(KeychainConstants.kCurrentSemester) as? Semester {
-            let params = [UserConstants.kSemesterID: currentSemester.id]
```

Comment:
you wanna add this into params if filter is nil, cuz we want to by default just return the mentors of the current semester